src/gsy_e/gsy_e_core/simulation/__init__.py

```diff
@@ -0,0 +1,521 @@
+"""
```

Comment:
We've previously talked about avoiding putting code inside `__init__.py` modules. I'd suggest moving all this code into a `simulation.py` module for clarity and to follow on that.

Reply:
I just noticed @spyrostz's [comment here](https://github.com/gridsingularity/gsy-e/pull/1533#pullrequestreview-1109126282):
> [...] avoids changing the imports of the Simulation class (since it will be the same as now, from `gsy_e.gsy_e_core.simulation import Simulation`) [...]

This can be achieved by importing inside of `__init__.py` just the classes you want to expose on the outside. For example:
```py
# __init__.py
from gsy_e.gsy_e_core.simulation.simulation import Simulation

__all__ = [
    "Simulation"
]
```
So this will preserve the possibility to just use
```py
from gsy_e.gsy_e_core.simulation import Simulation
```
as before.

Reply:
OK, will do.